lesson04.ipynb

```diff
@@ -0,0 +1,1158 @@
+{
```

Comment:
<p>A concerete example here would be good. Perhaps</p><ul><li>Time spent on website and prob of checking out</li><li>Amount eaten at buffet and whether someone regrets it or not</li><li>Fraudulent activity on a credit card, where low dollar values are test transactions, and really high ones are someone trying to clean out a card</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bwengals/etc/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>added the 1st one. the second one is a little too real</p>

Reply:
:D